docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,163 @@
+
+Since the setup is the same as in the `Garden` cluster, see details in 'Access to VictoriaLogs in the `Garden` cluster'.
+
+### 3. Deploy VictoriaLogs in Shoot Control-Planes
+
+During the `Shoot` reconciliation flow, the new `VictoriaLogs` component will be deployed to the Control-Plane namespace of `Shoot` clusters. The operator will manage the resulting `VLSingle` CustomResource and create a `VictoriaLogs` statefulset.
+
+Example VLSingle CR:
+```
+apiVersion: operator.victoriametrics.com/v1
+kind: VLSingle
+metadata:
+  name: example
+  namespace: shoot--local--local
+spec:
+  storage:
+    resources:
+      requests:
+        storage: 30Gi
+  resources:
+    requests:
+      memory: 100Mi
+      cpu: 10m
+    limits:
+      memory: 300Mi
+      cpu: 50m
+```
+
+#### Access to VictoriaLogs in the `Shoot` Control-Plane
+
+Access to Vali is exposed via an ingress in the shoot control-plane. This will continue with VictoriaLogs as well. Log shippers will need to be configured via OTLP.
+
+### 4. Migration from Vali to VictoriaLogs
+
+#### Data Migration
+During all the replacements of `Vali` we want to ensure that existing clusters do not get disrupted. For this reason, 3 strategies for migration have been considered:
+- "One-shot" migration. Reformat all the Vali data into the VictoriaLogs format (or OTLP). Could be done via the Collector since there is no tool that exists to do this OOTB. This would include:
+    Pros:
+    - Migrations step is quick.
+    - No friction during the migration period since there is only one source of logs after the migration.
+    Cons:
+    - Development of the necessary tooling to do the migration.
+    - Risk of data loss if something goes wrong during the migration.
+- "Rolling" migration. Get the VictoriaLogs instances up and reroute the logs to them without removing Vali. This would require both of them to be accessible for the whole rotation period of the logs and it would introduce friction for the time being since there would be 2 sources of logs.
+    Pros:
+    - No risk of data loss.
+    - Easy to setup when using an OpenTelemetry Collector since it supports multiple backends.
+    Cons:
+    - Operators would need to access 2 sources of logs until the rotation period is over. Vali (Plutono) for the logs before the migration point and VictoriaLogs (VictoriaUI) for the logs after the migration point.
+- "Dual" migration. Same thing as the "rolling" strategy but logs are sent to both backends simultaneously. This would remove the friction but would incur a higher storage cost due to the replication of the logs on both backends.
+    Pros:
+    - No risk of data loss.
+    - No friction during the migration period since all logs are visible in both backends.
+    Cons:
+    - Higher storage cost due to the replication of the logs on both backends.
+
+All migration strategies are viable and the choice of which one to use will depend on the specific characteristics of the landscape.
```

Comment:
How will the strategy be chosen for a particular landscape? 
When a strategy is chosen for a landscape, do you mean that all clusters (garden, all seeds and all shoots) will follow the same strategy? 
What are the landscapes characteristics that would imply a specific migration? And finally, isn't it better to enforce a single migration strategy instead of having to support all cases?